app/src/components/MultipleSelectionDisplay.tsx

```diff
@@ -14,6 +14,7 @@ export default function MultipleSelectionDisplay(props: Props) {
     // TODO: don't use index?
     const selectionDisplays = props.selections.map((sel, i) => (
         <SelectionDisplay
+            key={i}
```

Comment:
Same reason.

Reply:
We could save an ID on creating a `Selection` using uuid().

You must save it. Just putting a new `uuid()` as a key every time doesn't save us re-rendering.

Reply:
Write up an issue for this. I think it would be useful to re-render all `SelectionDisplay`s when we add/remove one.